alternator/server.cc

```diff
@@ -211,9 +212,28 @@ class local_nodelist_handler : public gated_handler {
         // using _gossiper().get_live_members(). But getting
         // just the list of live nodes in this DC needs more elaborate code:
         auto& topology = _proxy.get_token_metadata_ptr()->get_topology();
-        sstring local_dc = topology.get_datacenter();
-        std::unordered_set<gms::inet_address> local_dc_nodes = topology.get_datacenter_endpoints().at(local_dc);
+        // /localnodes lists nodes in a single DC. By default the DC of this
```

Comment:
Where is /localnodes documented?

Reply:
I see it's not. Please follow up with documentation.

Reply:
/localnodes was originally documented in the separate Alternator Load Balancing design document, https://docs.google.com/document/d/1twgrs6IM1B10BswMBUNqm7bwu5HCm47LOYE-Hdhuu_8/edit - but this is no longer visible outside the ScyllaDB corporate, so definitely, I need to document it somewhere in docs/alternator. I'll create an issue to do it.

Reply:
https://github.com/scylladb/scylladb/issues/20989
I'll do it tomorrow.